src/Categories/Functor/Instance/Sets.agda

```diff
@@ -0,0 +1,23 @@
+{-# OPTIONS --safe #-}
+module Categories.Functor.Instance.Sets where
+
+open import Level
+open import Categories.Functor using (Endofunctor)
+open import Categories.Category.Instance.Sets
+open import Relation.Binary.PropositionalEquality as ≡ using (_≡_)
+
+open import Function using (id)
+open import Categories.Category
+import Data.List as List
+import Data.List.Properties
+
+ListFunctor : ∀ {o} → Endofunctor (Sets o)
+ListFunctor = record
+  { F₀ = List.List
+  ; F₁ = List.map
+  ; identity = λ {_} {x} → map-id x
+  ; homomorphism = λ {_} {_} {_} {_} {_} {x} → map-compose x
+  ; F-resp-≈ = λ {_} {_} {_} {_} f≈g {x} → map-cong (λ x → f≈g {x}) x
```

Comment:
is there a utility function to convert pointwise equality from explicit to implicit and vice versa? this feels really ugly

Reply:
[`_$-`](http://agda.github.io/agda-stdlib/Function.Base.html#2399) perhaps?